### PR TITLE
Fix Manifest for GP prior example

### PR DESCRIPTION
**Pull request by willtebbutt**
As discussed in #339

### CODE REVIEW

**Comment by willtebbutt** [comment]
Test failure is entirely unrelated to this PR. Not sure what's causing it, but it's a separate issue.

**Review by theogf — APPROVED** [review]

**Comment by willtebbutt** [comment]
Hmm I don't appear to be able to merge. Possibly because the FBM kernel AD tests are being annoying?

File: examples/gaussian-process-priors/Manifest.toml [hunk 1]
```diff
@@ -281,9 +281,9 @@ version = "2.1.0+0"
 
 [[KernelFunctions]]
 deps = ["ChainRulesCore", "Compat", "CompositionsBase", "Distances", "FillArrays", "Functors", "LinearAlgebra", "Random", "Requires", "SpecialFunctions", "StatsBase", "StatsFuns", "TensorCore", "Test", "ZygoteRules"]
-git-tree-sha1 = "e8b5ba31b6d18695fd46bfcd8557682839023195"
+path = "../.."
 uuid = "ec8451be-7e33-11e9-00cf-bbf324bd1392"
-version = "0.10.6"
+version = "0.10.8"
```
Comment (st--) [inline examples/gaussian-process-priors/Manifest.toml:286]: will we have to keep bumping this version and keep it in sync with actual package version ?

**Review by st-- — APPROVED** [review]

**Comment by st--** [comment]
@willtebbutt we should also update the examples/README.md instructions for adding new examples, do you want to just add that in here ?

File: examples/gaussian-process-priors/Manifest.toml (hunk 1 above)
Comment (willtebbutt) [reply examples/gaussian-process-priors/Manifest.toml:286]: Hmmm it seems to be fine running it locally -- I think we might just want to add a preamble to each of the example scripts along the lines of
```julia
using Pkg
Pkg.activate(@__DIR__)
Pkg.resolve()
Pkg.instantiate()
```
which should sort it out.

Comment (willtebbutt) [reply examples/gaussian-process-priors/Manifest.toml:286]: I'm going to leave this for now actually -- it seems to run fine locally when the KernelFunctions version is out of sync, provided that it's dev-ed correctly.

File: examples/README.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ From a Julia REPL started in your example script's directory, you can run
 ```julia
 julia> ] activate .
 julia> ] add Literate
-julia> ] add KernelFunctions
+julia> ] dev ../..
```
Comment (st--) [inline examples/README.md:47]:
```suggestion
julia> # the following line adds a dependency on KernelFunctions that is based on the local directories, not a hash:
julia> ] dev ../..
```

**Review by st-- — APPROVED** [review]

**Pull request merged**